client_test.go

```diff
@@ -1112,7 +1112,7 @@ func TestClient_Invoke_HTTP(t *testing.T) {
 	}
 	s := http.Server{Handler: handler}
 	go func() {
-		if err = s.Serve(l); err != nil && err != http.ErrServerClosed {
+		if err = s.Serve(l); err != nil && !errors.Is(err, http.ErrServerClosed) {
```

Comment:
Good good.  We should keep our eyes open to make sure we always use this method of comparison in the future!